src/backend/src/prisma/schema.prisma

```diff
@@ -122,14 +124,30 @@ model Change_Request {
 }
 
 model Scope_CR {
-  scopeCrId       Int            @id @default(autoincrement())
-  changeRequestId Int            @unique
-  changeRequest   Change_Request @relation(fields: [changeRequestId], references: [crId])
-  what            String
-  why             Scope_CR_Why[]
-  scopeImpact     String
-  timelineImpact  Int
-  budgetImpact    Int
+  scopeCrId        Int                @id @default(autoincrement())
+  changeRequestId  Int                @unique
+  changeRequest    Change_Request     @relation(fields: [changeRequestId], references: [crId])
+  what             String
+  why              Scope_CR_Why[]
+  scopeImpact      String
+  timelineImpact   Int
+  budgetImpact     Int
```

Comment:
Yes we should be removing these. Manual migration would be to convert all of the existing fields here being removed into a single proposed solution for each cr

Reply:
we can't remove them right away. we need to have both for now, then run a manual script on prod, make sure that nothing is using these old ones, then we can finally delete them in a later ticket. (just my opinion but to me it's the safest)

Reply:
sounds good if that's what you wanna do. me personally I would probably extract all the data first, turn it into a manual db script I can run on the new api, apply migration dropping all that data, and then run the manual script to put it back. but that's just me. I don't really care how it's done as long as it settles to where it needs to be. just use your best judgment 👍

Reply:
hmmm ok